app/routes/api.js

```diff
@@ -169,4 +170,92 @@ router.post('/report', (req, res) => {
     res.redirect('back');
 });
 
+// API endpoint to delete an attachment for a given attachment ID
+router.delete('/attachment/:attachmentID', (req, res, next) => {
+    res.setHeader('Content-Type', 'application/json')
+    try {
+        const attachmentID = req.params.attachmentID;
+
```

Comment:
Pretty much same as above :)
```suggestion
        const attachmentID = parseInt(req.params.attachmentID);

		if (isNaN(attachmentID)) {
			return res.status(404).send({ message: 'Invalid attachment ID in request.' });
		}
```